spec/models/time_period_spec.rb

```diff
@@ -51,14 +51,19 @@
   describe 'model methods' do
     before(:each) do
       TimePeriod.destroy_all
+      allow(ENV).to receive(:[]).with('START_WEEK_DAY').and_return('monday')
```

Comment:
you also can use `stub_const('ENV', {'START_WEEK_DAY' => 'monday'})` but your version is good too